src/jspin.c

```diff
@@ -212,10 +212,16 @@ Pin jshGetPinFromVarAndUnLock(JsVar *pinv) {
   // Whether a pin's state has been set manually or not
 BITFIELD_DECL(jshPinStateIsManual, JSH_PIN_COUNT); // TODO: This should be set to all 0
 
+/**
+ * Determine if the state of a given pin has been defined as being manually set.
```

Comment:
Please can you either take this out, use the existing docs that are in the header file, or update the docs in the header file to say the same thing?

This function should already be documented in Doxygen (and Eclipse's tooltips) because of the comments in the header file - it's crazy having *two different sets of comments* - especially when the existing ones (that will presumably get overwritten) elaborate more on what 'manually set' is:

https://github.com/espruino/Espruino/blob/master/src/jspin.h#L207

Sorry to keep banging on about this, but it's wasting everyone's time and is actually making the source **harder to understand**.